packages/core/getBusyTimes.ts

```diff
@@ -55,7 +76,19 @@ export async function getBusyTimes(params: {
   if (credentials?.length > 0) {
     const calendarBusyTimes = await getBusyCalendarTimes(credentials, startTime, endTime, selectedCalendars);
 
-    busyTimes.push(...calendarBusyTimes); /*
+    busyTimes.push(
+      ...calendarBusyTimes.map((value) => ({
+        ...value,
+        end: dayjs(value.end)
```

Comment:
Always add beforeEventBuffer to the end time so we get separation on calendar events 

Without this we could book a meeting straight after a normal calendar event.